### PR TITLE
Update issue templates

### LINKED ISSUE #225: create templates for issues and pull requests

### PR DESCRIPTION
**Pull request by garciajordy**
- Add template for issues.
- Add template for PR.

fix #225

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+**Controllers:**
+- Add the changes you did to the controller.
+- Add anything else you did to the model.
+
+### Rspec
+**FactoryBot:**
+- Add the new factories you created.
+- Add anything else you did to the factories.
+
+**ModelTesting:**
```
Comment (edimossilva) [inline docs/pull_request_template.md:16]: maybe should add checklist with options test model/test  request/test feature/  tests not required?

File: .github/ISSUE_TEMPLATE/custom.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+labels: ''
+assignees: ''
+
+---
+
+<!-- Please, describe what this issue is about and any proposed changes. -->
+
+## Todo
+
+- [ ] I have added this issue to the [Workflow](https://github.com/nebulab/playbook/projects/1) GitHub project.
```
Comment (edimossilva) [inline .github/ISSUE_TEMPLATE/custom.md:14]: nebulab?

File: .github/ISSUE_TEMPLATE/custom.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
```
Comment (edimossilva) [inline .github/ISSUE_TEMPLATE/custom.md:5]: Label should be done used the github labels right? is not good for the user to have to type the labels (same for assigned)?

File: .github/ISSUE_TEMPLATE/custom.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
```
Comment (edimossilva) [inline .github/ISSUE_TEMPLATE/custom.md:4]: What is the diff between the name to the title?

File: .github/ISSUE_TEMPLATE/custom.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```
Comment (edimossilva) [inline .github/ISSUE_TEMPLATE/custom.md:3]: what is the difference between the `about` session and the description on line 10?

**Review by edimossilva — CHANGES_REQUESTED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (garciajordy) [reply docs/pull_request_template.md:16]: I was thinking about it but what if you don't have to do any testing you can just delete the lines I wrote but I can add it

File: .github/ISSUE_TEMPLATE/custom.md (hunk 2 above)
Comment (garciajordy) [reply .github/ISSUE_TEMPLATE/custom.md:14]: oops

Comment (garciajordy) [reply .github/ISSUE_TEMPLATE/custom.md]: fixed

File: docs/pull_request_template.md (hunk 1 above)
Comment (garciajordy) [reply docs/pull_request_template.md:16]: done

File: .github/ISSUE_TEMPLATE/custom.md (hunk 3 above)
Comment (garciajordy) [reply .github/ISSUE_TEMPLATE/custom.md]: done

File: .github/ISSUE_TEMPLATE/custom.md (hunk 4 above)
Comment (garciajordy) [reply .github/ISSUE_TEMPLATE/custom.md]: It was automatic through Github I modified it.

File: .github/ISSUE_TEMPLATE/custom.md (hunk 5 above)
Comment (garciajordy) [reply .github/ISSUE_TEMPLATE/custom.md]: Related to previous comment

File: docs/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- Add tests you did to the model.
+
+**Requests:**
+- Add tests you did to the requests.
+
+fix #issuenumber
```
Comment (edimossilva) [inline docs/pull_request_template.md:22]: I think this should be the first line

File: docs/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+### Rspec
+**FactoryBot:**
+- Add the new factories you created.
+- Add anything else you did to the factories.
+
+**ModelTesting:**
+- Add tests you did to the model.
+
+**Requests:**
```
Comment (edimossilva) [inline docs/pull_request_template.md:19]: if you have model and requests here, you should also have system tests as well

**Review by edimossilva — CHANGES_REQUESTED** [review]

**Review by edimossilva — APPROVED** [review]
Niceeee, squash and merge!

**Pull request merged**